antarest/study/business/areas/hydro_management.py

```diff
@@ -138,13 +163,22 @@ def set_field_values(
         """
         commands: List[UpdateConfig] = []
 
+        hydro_config = self.get_hydro_config(study)
+
+        # create a reformatted hydro config dictionary so the fields name can match `FIELDS_INFO` fields name
+        reformat_hydro_config = {}
+        for key, value in hydro_config.items():
+            reformat_hydro_config[key.replace("-", "_").replace(" ", "_")] = value
```

Comment:
Do you really need those three lines ? I removed them and the unit tests still passes